### PR TITLE
Fix i2cscan trying to begin in slave mode

### PR DESCRIPTION
**Pull request by sctanf**
~Not working, causes esp to crash(?) mid-scan~
Works fine with only i2c wires attached

### CODE REVIEW

File: lib/i2cscan/i2cscan.cpp [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #include "i2cscan.h"
 
 #ifdef ESP8266
-uint8_t portArray[] = {16, 5, 4, 2, 14, 12, 13};
+int portArray[] = {16, 5, 4, 2, 14, 12, 13};
```
Comment (TheDevMinerTV) [inline lib/i2cscan/i2cscan.cpp:4]: explain your reason to change this, there's no pins < 0

File: lib/i2cscan/i2cscan.cpp [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 String portMap[] = {"D0", "D1", "D2", "D4", "D5", "D6", "D7"};
 #elif defined(ESP32)
-uint8_t portArray[] = {4, 13, 14, 15, 16, 17, 18, 19, 21, 22, 23, 25, 26, 27, 32, 33};
+int portArray[] = {4, 13, 14, 15, 16, 17, 18, 19, 21, 22, 23, 25, 26, 27, 32, 33};
```
Comment (TheDevMinerTV) [inline lib/i2cscan/i2cscan.cpp:7]: same here.

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]
found these issues things

**Review by TheDevMinerTV — APPROVED** [review]

File: lib/i2cscan/i2cscan.cpp [hunk 3]
```diff
@@ -81,6 +81,7 @@ namespace I2CSCAN
                 Serial.println(address, HEX);
             }
         }
+        Wire.end();
```
Comment (unlogisch04) [inline lib/i2cscan/i2cscan.cpp:84]: Wire.end(); does not exist in ESP8266
`#ifdef ESP32`
`Wire.end()`
 `#endif`

**Comment by sctanf** [comment]
Not an issue since trackers are using espressif32 v3.5.0 for now

**Pull request closed without merging**